invenio_saml/invenio_accounts/utils.py

```diff
@@ -17,12 +19,11 @@
 # from .models import User
 from invenio_accounts.models import User
 from invenio_db import db
+from invenio_oauthclient.errors import AlreadyLinkedError
```

Comment:
Most probably the functions were copied/pasted to avoid to depend on `invenio-oauthclient`.
If we add these imports, we will need to add to the `setup.py` the dependency `invenio-oauthclient`.

Reply:
the module is already depending on `invenio-oauthclient` see : https://github.com/inveniosoftware/invenio-saml/blob/1e0714682f076d5e2aecb341a5095d5c18bd8cf3/setup.py#L24

Reply:
I missed that! Thanks a lot then, perfect!